pnpm-lock.yaml

```diff
@@ -0,0 +1,17779 @@
+lockfileVersion: 5.3
```

Comment:
Should also be in gitignore since we' re using npm not yarn

Reply:
yes these will be removed